docs/UserGuide.md

```diff
@@ -336,19 +358,37 @@ You can edit an existing student's fields in F.A.K.E.J.A.R.V.I.S.
 
 Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [th/TELEGRAM_HANDLE] [t/TAG]…​`
 
-[//]: # (* Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed 
-person list. The index **must be a positive integer** 1, 2, 3, …​   THIS CAN BE IN THE GLOSSARY)
+<div class="alert alert-info"> :information_source: Note: 
+
+* Edits the person at the specified `INDEX`. 
+* The index refers to the index number shown in the displayed person list. 
+* The index **must be a positive integer** 1, 2, 3, …​
+* At least one of the optional fields must be provided. (i.e NAME, PHONE, EMAIL, TELEGRAM_HANDLE)
+* Editing tags overwrites existing tags; it's not cumulative. To remove all tags, use `t/` without specifying any tags after it.
+* You can remove all the person’s tags by typing `t/` without specifying any tags after it.
```

Comment:
Perhaps remove all tags should not be repeated? I think it should be in a line of its own.